src/components/Disturber.tsx

```diff
@@ -78,69 +79,76 @@ export const Disturber = ({ navigation, publicJsonFile }: Props) => {
 
   if (!isVisible || !closestItem) return null;
 
+  const {
+    backgroundColor,
+    button,
+    description,
+    headline,
+    picture,
+    showButtonToClose = true,
+    title
+  } = closestItem;
+  const { title: buttonTitle, navigationTo, params } = button;
```

Comment:
if button is undefined, this could crash i guess.
can you check that please. because if button is not an object, destruction fails.